integration/src/integration/java/pl/allegro/tech/hermes/integration/helper/Waiter.java

```diff
@@ -79,20 +77,47 @@ public void untilSubscriptionMetricsIsRemoved(TopicName topicName, String subscr
         untilZookeeperNodeDeletion(zookeeperPaths.subscriptionMetricsPath(topicName, subscriptionName), zookeeper);
     }
 
-    public void untilSubscriptionIsActivated(Topic topic, String subscription) {
-        untilSubscriptionHasState(topic, subscription, Subscription.State.ACTIVE);
+    public void untilSubscriptionIsActivated(long currentTime, Topic topic, String subscription) {
+        until(Duration.ONE_MINUTE, topic, subscription, sub -> sub.getSignalTimesheet().getOrDefault(START, 0L) > currentTime);
+    }
 
-        waitAtMost(adjust(Duration.TEN_SECONDS)).until(() -> {
-            return endpoints.consumer().listSubscriptions().contains(topic.getQualifiedName() + "$" + subscription);
-        });
+    public void untilSubscriptionIsActivated(Topic topic, String subscription) {
+        untilSubscriptionIsActivated(clock.millis(), topic, subscription);
     }
 
     public void untilSubscriptionIsSuspended(Topic topic, String subscription) {
         untilSubscriptionHasState(topic, subscription, Subscription.State.SUSPENDED);
 
-        waitAtMost(adjust(Duration.TEN_SECONDS)).until(() -> {
-            return !endpoints.consumer().listSubscriptions().contains(topic.getQualifiedName() + "$" + subscription);
-        });
+        waitAtMost(adjust(Duration.ONE_MINUTE)).until(() ->
```

Comment:
why is base for adjust one minute? it means after adjusting we will be waiting 3 minutes for subscription to suspend, with so many tests - i don't know if thats a good idea